### PR TITLE
Fixed ignore block listener

### PR DESCRIPTION
**Pull request by Mingela**

### CODE REVIEW

File: eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt [hunk 1]
```diff
@@ -95,7 +98,7 @@ class EthChainListener(
      */
     private fun publishEthBlockAndSaveHeight(ethBlock: EthBlock) {
         ethBlocksSubject.onNext(ethBlock)
-        val height = ethBlock.block.number
+        val height = ethBlock.block.number.add(bigIntOne)
```
Comment (Alexey-N-Chernyshov) [inline eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt]:
```suggestion
        val height = ethBlock.block.number.inc()
```

Comment (Mingela) [reply eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt]: tochnyak

File: eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt [hunk 2]
```diff
@@ -109,5 +112,7 @@ class EthChainListener(
     /**
      * Logger
      */
-    companion object : KLogging()
+    companion object : KLogging() {
+        val bigIntOne: BigInteger = BigInteger.ONE
```
Comment (Alexey-N-Chernyshov) [inline eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt]: // no need if incremented with `inc()`

File: eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt (hunk 1 above)
Comment (Mingela) [reply eth/src/main/kotlin/com/d3/eth/sidechain/EthChainListener.kt]: But it will lead to creation of BigInt object every time btw

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Pull request merged**